server/src/main/java/org/elasticsearch/action/admin/cluster/stats/TransportClusterStatsAction.java

```diff
@@ -313,4 +314,67 @@ protected boolean isFresh(Long currentKey, Long newKey) {
             return newKey <= currentKey;
         }
     }
+
+    public static final class AdditionalStats {
```

Comment:
As far as I can see, this class is never used outside its parent - does it need to be public? Package-private seems to work here.

Reply:
It's visible in the type signature of `TransportClusterStatsAction` which itself is public. At least, I get warnings if I make it package-private.